cogs/info.py

```
@@ -116,7 +116,9 @@ async def about(self, i: Interaction):
             value=f"> CPU usage: {psutil.cpu_percent(interval=1)}% \n> RAM usage: {round(mem)}MB",
             inline=False
         )
-        cogs_dir = '/home/ubuntu/bots/python/pluto/cogs'
+        # derive project root (…/pluto) dynamically
+        BASE_DIR = pathlib.Path(__file__).resolve().parent.parent
```

Ruff (0.11.9)

Undefined name `pathlib`

(F821)